hazelcast/src/main/java/com/hazelcast/cache/HazelcastCachingProvider.java

```diff
@@ -106,15 +108,15 @@
  * <h3>Examples</h3>
  * <p><b>Obtain a member-side caching provider backed by an existing HazelcastInstance.</b>
  * In this example the member-side caching provider is selected by setting the value of
- * system property {@code hazelcast.jcache.provider.type} to value "{@code server}". An existing
+ * system property {@code hazelcast.jcache.provider.type} to value "{@code member}". An existing
  * {@code HazelcastInstance} is referenced by instance name in the {@code Properties} provided as
  * argument to {@link CachingProvider#getCacheManager(URI, ClassLoader, Properties)}.
  * <blockquote><pre>
  * Config config = new Config();
  * config.setInstanceName("hz-jcache");
  * HazelcastInstance member = Hazelcast.newHazelcastInstance(config);
  *
- * System.setProperty("hazelcast.jcache.provider.type", "server");
+ * System.setProperty("hazelcast.jcache.provider.type", "member");
```

Comment:
Doesn't this break the backward compatibility?

Reply:
"server" still works, see https://github.com/hazelcast/hazelcast/pull/17320/files#diff-0041c466c89dab74d5d3d0ade5b97e63R56 and https://github.com/hazelcast/hazelcast/pull/17320/files#diff-0041c466c89dab74d5d3d0ade5b97e63R200